### ISSUE TITLE
HOT validation: In response Mexican earthquake - Puebla

### ISSUE THREAD

**Issue by piligab** [developmentseed/mapping#1]
On Tuesday 19 September 2017, a powerful earthquake (magnitude 7.1) jolted central Mexico, cracking building facades and scattering rubble on streets in the capital on the anniversary of a devastating 1985 quake. Its epicenter was near the Puebla state town of Raboso, about 76 miles southeast of Mexico City.

Development Seed data team as part of the HOT community is taking part of this disaster mapping efforts, We are going to validate the buildings and highways as the task describe!

### Tasking Manager 
- [x] https://tasks.hotosm.org/project/3603

### Mapping sources
- Bing imagery 
- JOSM 

#### Task Coverage 
- Puebla (city center), Puebla

#### Activities
- Go through the instructions before you start. 
- Map and valid all the roads and buildings in the task.

#### - Task #3603
![image](https://user-images.githubusercontent.com/19536044/34878087-dd336438-f776-11e7-97f2-c8c0d26c6946.png)

- Right now, the task is 75% mapped and 3% validated. So let's map and valid all the roads and buildings at 100%. 
![image](https://user-images.githubusercontent.com/19536044/34878567-94c1d6ce-f778-11e7-9ba8-49866270dfce.png)

if you have any comment or suggestion please post on this ticket!

cc. @olafveerman @Rub21

**Comment by olafveerman** [comment]
Thanks @piligab 
Pulling in the Data Team as well: @developmentseed/data-team

**Comment by karitotp** [comment]
On Friday, our team validated 535 tasks, bringing the validation to **76%**. In general the community did a good job, all the buildings and roads were mapped and most of them were of good quality. 


![screenshot from 2018-01-12 21-27-49](https://user-images.githubusercontent.com/11504548/34911685-69df0578-f89d-11e7-9af5-a0da9ea5ad46.png)
_76% in validation_

However, some occurring issues  were: 
- Building roofs were mapped instead of the base.
- Crossing building.
- Overlapping between footprints and other features.
- Some buildings had to be split into multiple buildings. 

The main difficulty was the low quality of the imagery in the Eastern part to see the outline of the buildings.

Over the course of the next couple of days, our team will continue to be active in the tasks to map and validate the remaining tasks.

![screenshot from 2018-01-13 20-20-53](https://user-images.githubusercontent.com/11504548/34911786-55d774d2-f89f-11e7-8347-c1fd9ea5fc34.png)
_Currently status of the tasking_

**Comment by mikelmaron** [comment]
@karitotp great progress validating and good write up of what you found. A few questions

* Is the team providing feedback directly to the mappers on changeset comments or in the tasking manager?
* What was the percentage of problems found?
* Did the data team employ any special techniques to make validation faster and more effective?
* What's your thinking on sharing back what you found with the broader HOT and OSM community?

**Comment by karitotp** [comment]
@mikelmaron concerning to your questions

> Is the team providing feedback directly to the mappers on changeset comments or in the tasking manager?

We sent direct messages to some users with positive feedback about their good job in mapping. 
> What was the percentage of problems found?

According the buildings that were fixed, we can estimate an average between 25% and 30%.
> Did the data team employ any special techniques to make validation faster and more effective?

The size of the boxes was split into small areas, there were cases in which to improve or/and correct an edition we download a bit more, so also the nearby buildings were fixed, that is how we moved faster. Also use the JOSM validator is a useful tool to find common errors like crossing building.

In general We consider that the good job of the mappers did easier the validation. 

> What's your thinking on sharing back what you found with the broader HOT and OSM community?

We post this to share with the community about our experience during the validation work. As we know, validation is an important part of Hot's tasks to ensure the quality of added data. We also share our findings to let know the comunity about the common errors that can occur during the mapping and provide some feedbacks.
